traffic_ops/testing/api/v14/deliveryservice_required_capabilities_test.go

```diff
@@ -0,0 +1,221 @@
+package v14
+
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/lib/go-util"
+)
+
+func TestGetDeliveryServiceRequiredCapabilities(t *testing.T) {
```

Comment:
This should be called `TestDeliveryServiceRequiredCapabilities` omit the GET in the name

Reply:
why?

Reply:
Because you are running create and delete as well via the with objs

Reply:
Because you are running create and delete as well via the with objs